### ISSUE TITLE
Unintentional illegal pun?

### ISSUE THREAD

**Issue by dlutz2** [ufbmi/geographical-entity-ontology#18]
In nation-and-dependency-instances.owl, the _is in aggregate_ (http://purl.obolibrary.org/obo/GEO_000000328) and _is subordinate authority of_ (http://purl.obolibrary.org/obo/GEO_000000329) properties are re-declared as annotation properties, resulting in an illegal pun. Assume that was unintentional?

**Comment by hoganwr** [comment]
Yes, will attempt a fix.  A maddening problem a number of years ago, which
obviously we haven't managed to fully expurgate,

**Comment by dlutz2** [comment]
Found 2 more (I think that's all of them)
bodies-of-water.owl re-declares:
    _adjacent to_ - http://purl.obolibrary.org/obo/RO_0002220
    _located in_ - http://purl.obolibrary.org/obo/RO_0001025
as annotation properties
